manubot/util.py

```diff
@@ -142,3 +143,35 @@ def read_serialized_dict(path: str) -> dict:
         f"Expected data encoded by {path!r} to be a dictionary at the top-level. "
         f"Received {data.__class__.__name__!r} instead."
     )
+
+
+def _yaml_str_representer(dumper: "yaml.Dumper", data: str):
+    """
+    Use YAML literal block style for multiline strings.
```

Comment:
How universal is this literal block style?

Reply:
I think any parser that fully implements the YAML 1.1 spec should be able to parse this.

At https://yaml-multiline.info/, the format we specify for multiline strings, matches the options under block scalars:

- Block Scalar Style: Keep newlines (literal)
- Block Chomping: No newline at end (strip)

From what I can tell, this format is the best if we envision the YAML being tracked with git.